utils/docker/Dockerfile.centos.8

```diff
@@ -218,11 +218,13 @@ USER daos_server:daos_server
 # select compiler to use
 ARG COMPILER=gcc
 ARG JOBS=$DEPS_JOBS
+ARG DAOS_BUILD_TYPE=release
 ARG DAOS_BUILD=$DAOS_DEPS_BUILD
 
 # Build DAOS
 RUN [ "$DAOS_BUILD" != "yes" ] || \
-    { scons-3 --jobs $JOBS install PREFIX=/opt/daos COMPILER=$COMPILER && \
+    { scons-3 --jobs $JOBS install PREFIX=/opt/daos \
+	COMPILER=$COMPILER BUILD_TYPE=$DAOS_BUILD_TYPE && \
```

Comment:
Do you need to explicitly set TARGET_TYPE=release so you actually reuse the previously built dependencies?  The default TARGET_TYPE is 'default' which means use the BUILD_TYPE

Reply:
Yes, I was expecting to have to do this.  In place now.